src/collectd.conf.pod

```diff
@@ -4082,11 +4082,11 @@ If enabled, the plugin sends a notification if the replication slave I/O and /
 or SQL threads are not running. Defaults to B<false>.
 
 =item B<WsrepStats> I<true|false>
- 
+
```

Comment:
All these "unrelated changes" are simply a result of my editor removing trailing whitespace from lines on save.

I will be rebasing to tidy up this pull request and will make sure to remove these changes from my change set.